lib/handbook/addon/docs/template.hbs

```diff
@@ -24,6 +24,7 @@
         <nav.item @label='loading-indicator' @route='docs.components.loading-indicator' />
         <nav.item @label='new-project-modal' @route='docs.components.new-project-modal' />
         <nav.item @label='new-project-navigation-modal' @route='docs.components.new-project-navigation-modal' />
+        {{nav.item 'osf-button' 'docs.components.osf-button'}}
```

Comment:
We need to update the blueprint.

Reply:
Again.

Reply:
No, I think I just made this before you told me about the label and route params. I checked the test fixture for the blueprint and it's correct. _phew_